src/containers/Board/DetailBoard/emotion.ts

```diff
@@ -0,0 +1,33 @@
+import { Button, Row } from "@/components";
+import styled from "@emotion/styled";
+
+export const Container = styled.div`
+  width: 100vw;
+  height: 100vh;
+  flex-direction: column;
+  background-color: #fff;
+`;
+
+export const Title = styled.h2`
+  padding: 12px 20px;
+`;
+
+export const WrapBar = styled.div`
+  padding: 12px 20px;
+  flex-direction: row;
+  /* align-items: center; */
+  display: flex;
+  justify-content: flex-end;
+`;
```

Comment:
```suggestion
export const WrapBar = styled(Row)`
  padding: 12px 20px;
  flex-direction: row;
  /* align-items: center; */
  display: flex;
  justify-content: flex-end;
`;
```
flex display 사용하기 편하려고 Row와 Column이라는 컴포넌트 만들어놨어요 참고해주시면 좋을거같아요